### DISCUSSION TITLE
'killlargecrawler' mission is called "*A* large crawler" instead of a group of crawlers.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15163] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

I'm sure we've all noticed this for years now, but the mission "Killing a large crawler" is just plain wrong...
This typo is even in it's description, stating "An especially large crawler". 

It makes it sound like you're supposed to be fighting one big crawler ( presumably smaller than a broodmother but bigger than your standard crawler... )

I mean, it is shorter than saying "a large swarm of crawlers" but I don't think that's the reason since spineling swarms and mudraptor swarms are properly described in their missions. Seems like just an odd typo that was forgotten, maybe because the mission ID itself is also called "largecrawler" so it wasn't immedeately obvious to localization process.

### Reproduction steps

Find the mission 'killlargecrawler': "Killing a large crawler".

The mission title and description says its 'a large crawler' despite clearly being a group of crawlers in the mission. Literally unplayable.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
the mission does have a large crawler Crawler_large

**Reply by Regalis11** [reply]
Yeah, this doesn't seem like a bug. There's one large crawler accompanied by a handful of normal ones.
